internal/command/action.go

```diff
@@ -14,40 +16,42 @@ import (
 )
 
 type ActionConfig struct {
-	Args        []string
-	Provider    string
-	Filepath    string
-	YamlInline  string
-	Subs        map[string]string
-	Ignores     []string
-	IgnoreAll   bool
-	Environment string
+	Args           []string
+	Provider        string
+	Filepath        string
+	YamlInline      string
+	Subs            map[string]string
+	Ignores         []string
+	IgnoreAll       bool
+	Environment     string
+	AllProvVersions bool
```

Comment:
nit: `AllProvVersions` -> `ShowProviderVersions` as we probably have no need to be terse here.